implementations/rust/ockam/ockam_command/src/node/create/foreground.rs

```diff
@@ -46,18 +50,30 @@ impl CreateCommand {
             return Err(miette!("Node {} is already running", &node_name));
         };
 
+        let trust_options = opts
+            .state
+            .retrieve_trust_options(
+                &self.trust_opts.project_name,
+                &self.trust_opts.authority_identity,
+                &self.trust_opts.authority_route,
+                &self.trust_opts.credential_scope,
+            )
+            .await
+            .into_diagnostic()?;
+
+        // Create TCP transport
         let tcp = TcpTransport::create(ctx).await.into_diagnostic()?;
         let tcp_listener = tcp
             .listen(&self.tcp_listener_address, TcpListenerOptions::new())
             .await
             .into_diagnostic()?;
-
-        debug!(
-            "set the node {node_name} listener address to {:?}",
+        debug!(%node_name,
+            "listener address set to {:?}",
             tcp_listener.socket_address()
         );
 
         // Set node_name so that node can isolate its data in the storage from other nodes
```

Comment:
This is a hack that I'd like to fix at some stage because the `CliState` is not supposed to be node-specific and we could have data races down the line.

Reply:
Do we have an issue for that? I can open one so we don't forget

Reply:
I'm giving it a go right now since I'm kind of blocked.